lib/node_modules/@stdlib/math/base/special/cosm1/test/test.native.js

```diff
@@ -0,0 +1,131 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2018 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var resolve = require( 'path' ).resolve;
+var tape = require( 'tape' );
+var linspace = require( '@stdlib/array/base/linspace' );
+var isnan = require( '@stdlib/math/base/assert/is-nan' );
+var abs = require( '@stdlib/math/base/special/abs' );
+var cos = require( '@stdlib/math/base/special/cos' );
+var PINF = require( '@stdlib/constants/float64/pinf' );
+var NINF = require( '@stdlib/constants/float64/ninf' );
+var EPS = require( '@stdlib/constants/float64/eps' );
+var PI = require( '@stdlib/constants/float64/pi' );
+var tryRequire = require( '@stdlib/utils/try-require' );
+
+
+// VARIABLES //
+
+var NPIO4 = -7.85398163397448309616e-1; // -4/π
+var PIO4 = 7.85398163397448309616e-1; // 4/π
+var cosm1 = tryRequire( resolve( __dirname, './../lib/native.js' ) );
+var opts = {
+	'skip': ( cosm1 instanceof Error )
+};
+
+
+// FIXTURES //
+
+var data = require( './fixtures/python/data.json' );
+
+
+// TESTS //
+
+tape( 'main export is a function', opts, function test( t ) {
+	t.ok( true, __filename );
+	t.strictEqual( typeof cos, 'function', 'main export is a function' );
+	t.end();
+});
+
+tape( 'the function computes the cosine minues one more accurately inside the interval [-π/4,π/4]', opts, function test( t ) {
```

Comment:
Would you mind checking for and potentially fixing this typo also in the main `test.js` file, @gunjjoshi?

```suggestion
tape( 'the function computes the cosine minus one more accurately inside the interval [-π/4,π/4]', opts, function test( t ) {
```

Reply:
@Planeshifter Done!

Reply:
Thanks!